### PR TITLE
Fix "IPv6 connection unreliable from external PC"

### PR DESCRIPTION
**Pull request by joenpera**
Avahi workaround

Jira-Id: SECO-7551

### CODE REVIEW

**Review by saauvine — APPROVED** [review]

File: modules/sc-mesh-secure-deployment/src/nats/src/comms_service_refresh.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    def shutdown_service(self) -> None:
+        """
+        Shutdown the service
+        """
+        self.__event.set()
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger("CommsServiceRefresh")
+    comms_service_refresh = CommsServiceRefresh("mika-battlestation", logger)
```
Comment (johan-juntunen) [inline modules/sc-mesh-secure-deployment/src/nats/src/comms_service_refresh.py:55]: Maybe you can change it to some other name than "mika-battlestation" :)

**Review by johan-juntunen — CHANGES_REQUESTED** [review]
Some naming change request, otherwise looks good

File: modules/sc-mesh-secure-deployment/src/nats/src/comms_service_refresh.py (hunk 1 above)
Comment (joenpera) [reply modules/sc-mesh-secure-deployment/src/nats/src/comms_service_refresh.py:55]: 😂 that was for personal testing purposes and actually I removed it in mdmagent repo.  I will remove the main function.

Comment (joenpera) [reply modules/sc-mesh-secure-deployment/src/nats/src/comms_service_refresh.py]: [9e81465](https://github.com/tiiuae/mesh_com/pull/485/commits/9e81465dd3dfea8f88ddbf950b5d85c1ea3cfa78)

**Review by johan-juntunen — APPROVED** [review]
Looks good to me

**Review by saauvine — APPROVED** [review]

**Pull request merged**